flasc/utilities/lookup_table_tools.py

```diff
@@ -26,14 +30,19 @@ def get_yaw_angles_interpolant(
         end. This variable defaults to [10, 12], meaning that the yaw offsets are
         full at and below 10 m/s, then linearly transition to zero offsets
         at 12 m/s, and continue to be zero past 12 m/s. Defaults to [10, 12].
+        minimum_yaw_angle (float, optional): The minimum yaw angle in degrees.
+        Defaults to None.  If None, the minimum yaw angle is set to the minimum
+        yaw angle in the dataset.
+        maximum_yaw_angle (float, optional): The maximum yaw angle in degrees.
```

Comment:
This is probably nitpicking now, but there is also a question about whether the second line of an `Arg` description should be indented again, for example:
```
    maximum_yaw_angles (float, optional): The maximum yaw angle in degrees.
        Defaults to None.  If None, the maximum yaw angle is set to the maximum
        yaw angle in the dataset.
```
as opposed to the current
```
    maximum_yaw_angles (float, optional): The maximum yaw angle in degrees.
    Defaults to None.  If None, the maximum yaw angle is set to the maximum
    yaw angle in the dataset.
```
I think the extra indent does help with readability, but we aren't consistent throughout on this

Reply:
I agree I've tried to re-add the indent where I see it